admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -37,11 +37,17 @@ endif::[]
 ansible_ssh_user=ec2-user
 openshift_clusterid=mycluster
 ansible_become=yes
+openshift_cloudprovider_kind=aws <1>
+openshift_cloudprovider_aws_access_key=... <2>
```

Comment:
I meant to mention earlier, what do the `...` dots mean? Are these a reference to a variable that should be replaced with an AWS access key and AWS secret key by the user. If so, you could replace `...` with `<aws_access_key>` and `<aws_secret_key>`. This seems to be a well used format in the book for replaceables. Although, that topic is currently being reviewed.

Reply:
QA agreed to these changes, so I have incoporated them. Thanks again for your help.